vendor/bat-native-ledger/src/bat/ledger/internal/contribution/contribution.cc

```diff
@@ -437,42 +431,41 @@ void Contribution::OnEntrySaved(
     external_wallet_->Process(contribution_id, result_callback);
   }
 
-  if (queue->amount > 0) {
+  if ((*shared_queue)->amount > 0) {
     auto save_callback = std::bind(&Contribution::OnQueueSaved,
       this,
       _1,
       wallet_type,
       balance,
-      braveledger_bind_util::FromContributionQueueToString(queue->Clone()));
+      std::make_shared<ledger::ContributionQueuePtr>((*shared_queue)->Clone()));
```

Comment:
Can the shared pointer just be passed in to bind directly, without requiring the clone?

Reply:
fixed